modules/monix/src/weaver/monixcompat/suites.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package monixcompat
+
+import cats.effect.{ ConcurrentEffect, ContextShift, Resource, Timer }
+
+import monix.eval.Task
+import monix.eval.instances._
+import monix.execution.Scheduler
+
+trait BaseMonixSuite { self: ConcurrentEffectSuite[Task] =>
+  val ec                              = scala.concurrent.ExecutionContext.global
+  val scheduler                       = Scheduler(ec)
+  implicit def timer: Timer[Task]     = Task.timer(scheduler)
+  implicit def cs: ContextShift[Task] = Task.contextShift(scheduler)
+  implicit def effect: ConcurrentEffect[Task] =
+    new CatsConcurrentEffectForTask()(scheduler, Task.defaultOptions)
+}
+
+trait MutableMonixSuite extends MutableFSuite[Task] with BaseMonixSuite
```

Comment:
I'd call these `TaskSuite` for coherence, as the name of the effect type is `Task` (ie the cats-effect specific one is called `IOSuite` as opposed to `CatsSuite`)

Reply:
No problem. I was a bit confused as the ZIO suite is called `MutableZIOSuite`, and I thought the ZIO effect type was also called `Task`?

Reply:
You're not wrong, but `Task[R, A]` is type alias for `ZIO[R, Throwable, A]`, so the actual effect type is called `ZIO`

Reply:
Addressed in b644c8a823f102f8bcc638299adb09818008ed9c.